setup.py

```diff
@@ -50,8 +50,8 @@ def runSetup():
                 'gcs_oauth2_boto_plugin==1.9',
                 botoRequirement],
             'cwl': [
-                'cwltool==1.0.20170413194156',
-                'schema-salad==2.5.20170328195758',
+                'cwltool>=1.0.20170811195303',
+                'schema-salad>=2.6.20170806163416',
```

Comment:
Ugh, thanks @mr-c! ... I always wondered why Toil is hardpinning CWL so hard... since CWL 1.x, it doesn't break that often, does it? 

Every time I setup a new toil development environment I forget to tweak those back :/